android/java/org/chromium/chrome/browser/BraveSyncWorker.java

```diff
@@ -2378,7 +2378,7 @@ public void run() {
                         mBookmarkItems.add(bookmarkItem);
                     } else if (mAction.equals(DELETE_RECORD)) {
                         long defaultFolderId = (null != mDefaultFolder ? mDefaultFolder.getId() : 0);
-                        mBookmarkItems.add(BookmarkBridge.createBookmarkItem(id, BookmarkType.NORMAL, "", "", false,
+                        mBookmarkItems.add(BraveBookmarkModel.createBookmarkItem(id, BookmarkType.NORMAL, "", "", false,
```

Comment:
why do we need that change? As we change that method to public `makePublicMethod(sBookmarksBridgeClassName,"createBookmarkItem");`

Reply:
or it doesn't work that way for static?

Reply:
you can't do it that way here because it still has to build with the asm changes. So we create the method on the subclass to build it using javac and then in asm we make the superclass method public and remove the subclass method

Reply:
it also has the side-effect of having less "magic" from a dev perspective because you can see the methods and the comments in our code that reference the asm changes